2024-08-17 12:26:02,682 INFO    MainThread:45364 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-08-17 12:26:02,682 INFO    MainThread:45364 [wandb_setup.py:_flush():76] Configure stats pid to 45364
2024-08-17 12:26:02,683 INFO    MainThread:45364 [wandb_setup.py:_flush():76] Loading settings from C:\Users\Zain_\.config\wandb\settings
2024-08-17 12:26:02,683 INFO    MainThread:45364 [wandb_setup.py:_flush():76] Loading settings from c:\Users\Zain_\Dropbox\Personal\Data Science Projects\OpenAI_API_Fine_Tuning\wandb\settings
2024-08-17 12:26:02,683 INFO    MainThread:45364 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-08-17 12:26:02,683 INFO    MainThread:45364 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-08-17 12:26:02,683 INFO    MainThread:45364 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-08-17 12:26:02,683 INFO    MainThread:45364 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-08-17 12:26:02,683 INFO    MainThread:45364 [wandb_init.py:_log_setup():524] Logging user logs to c:\Users\Zain_\Dropbox\Personal\Data Science Projects\OpenAI_API_Fine_Tuning\wandb\run-20240817_122602-cbtejz2q\logs\debug.log
2024-08-17 12:26:02,683 INFO    MainThread:45364 [wandb_init.py:_log_setup():525] Logging internal logs to c:\Users\Zain_\Dropbox\Personal\Data Science Projects\OpenAI_API_Fine_Tuning\wandb\run-20240817_122602-cbtejz2q\logs\debug-internal.log
2024-08-17 12:26:02,683 INFO    MainThread:45364 [wandb_init.py:init():607] calling init triggers
2024-08-17 12:26:02,684 INFO    MainThread:45364 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-08-17 12:26:02,684 INFO    MainThread:45364 [wandb_init.py:init():632] re-initializing run, found existing run on stack: 1468wky1
2024-08-17 12:26:02,685 INFO    MainThread:45364 [wandb_run.py:_finish():2154] finishing run suspicious-cow-self/Reddit_Sentiment_Analysis/1468wky1
2024-08-17 12:26:02,685 INFO    MainThread:45364 [jupyter.py:save_history():444] not saving jupyter history
2024-08-17 12:26:02,685 INFO    MainThread:45364 [jupyter.py:save_ipynb():372] not saving jupyter notebook
2024-08-17 12:26:02,685 INFO    MainThread:45364 [wandb_init.py:_jupyter_teardown():452] cleaning up jupyter logic
2024-08-17 12:26:02,685 INFO    MainThread:45364 [wandb_run.py:_atexit_cleanup():2418] got exitcode: 0
2024-08-17 12:26:02,686 INFO    MainThread:45364 [wandb_run.py:_restore():2400] restore
2024-08-17 12:26:02,686 INFO    MainThread:45364 [wandb_run.py:_restore():2406] restore done
2024-08-17 12:26:05,106 INFO    MainThread:45364 [wandb_run.py:_footer_history_summary_info():4076] rendering history
2024-08-17 12:26:05,106 INFO    MainThread:45364 [wandb_run.py:_footer_history_summary_info():4108] rendering summary
2024-08-17 12:26:05,115 INFO    MainThread:45364 [wandb_run.py:_footer_sync_info():4035] logging synced files
2024-08-17 12:26:05,150 INFO    MainThread:45364 [wandb_init.py:init():657] starting backend
2024-08-17 12:26:05,150 INFO    MainThread:45364 [wandb_init.py:init():661] setting up manager
2024-08-17 12:26:05,153 INFO    MainThread:45364 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-08-17 12:26:05,153 INFO    MainThread:45364 [wandb_init.py:init():669] backend started and connected
2024-08-17 12:26:05,158 INFO    MainThread:45364 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-08-17 12:26:05,158 INFO    MainThread:45364 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-08-17 12:26:05,158 INFO    MainThread:45364 [wandb_init.py:init():764] updated telemetry
2024-08-17 12:26:05,212 INFO    MainThread:45364 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-08-17 12:26:06,321 INFO    MainThread:45364 [wandb_run.py:_on_init():2479] communicating current version
2024-08-17 12:26:06,374 INFO    MainThread:45364 [wandb_run.py:_on_init():2488] got version response 
2024-08-17 12:26:06,374 INFO    MainThread:45364 [wandb_init.py:init():848] starting run threads in backend
2024-08-17 12:26:11,469 INFO    MainThread:45364 [wandb_run.py:_console_start():2457] atexit reg
2024-08-17 12:26:11,470 INFO    MainThread:45364 [wandb_run.py:_redirect():2303] redirect: wrap_raw
2024-08-17 12:26:11,470 INFO    MainThread:45364 [wandb_run.py:_redirect():2368] Wrapping output streams.
2024-08-17 12:26:11,470 INFO    MainThread:45364 [wandb_run.py:_redirect():2393] Redirects installed.
2024-08-17 12:26:11,470 INFO    MainThread:45364 [wandb_init.py:init():891] run started, returning control to user process
2024-08-17 12:28:32,851 INFO    MainThread:45364 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-08-17 12:28:32,852 INFO    MainThread:45364 [wandb_setup.py:_flush():76] Configure stats pid to 45364
2024-08-17 12:28:32,852 INFO    MainThread:45364 [wandb_setup.py:_flush():76] Loading settings from C:\Users\Zain_\.config\wandb\settings
2024-08-17 12:28:32,852 INFO    MainThread:45364 [wandb_setup.py:_flush():76] Loading settings from c:\Users\Zain_\Dropbox\Personal\Data Science Projects\OpenAI_API_Fine_Tuning\wandb\settings
2024-08-17 12:28:32,852 INFO    MainThread:45364 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-08-17 12:28:32,852 INFO    MainThread:45364 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-08-17 12:28:32,852 INFO    MainThread:45364 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-08-17 12:28:32,852 INFO    MainThread:45364 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-08-17 12:28:32,852 INFO    MainThread:45364 [wandb_init.py:_log_setup():524] Logging user logs to c:\Users\Zain_\Dropbox\Personal\Data Science Projects\OpenAI_API_Fine_Tuning\wandb\run-20240817_122832-qtzjz9yt\logs\debug.log
2024-08-17 12:28:32,853 INFO    MainThread:45364 [wandb_init.py:_log_setup():525] Logging internal logs to c:\Users\Zain_\Dropbox\Personal\Data Science Projects\OpenAI_API_Fine_Tuning\wandb\run-20240817_122832-qtzjz9yt\logs\debug-internal.log
2024-08-17 12:28:32,853 INFO    MainThread:45364 [wandb_init.py:_jupyter_setup():470] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000200CE076690>
2024-08-17 12:28:32,853 INFO    MainThread:45364 [wandb_init.py:init():607] calling init triggers
2024-08-17 12:28:32,853 INFO    MainThread:45364 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-08-17 12:28:32,853 INFO    MainThread:45364 [wandb_init.py:init():632] re-initializing run, found existing run on stack: cbtejz2q
2024-08-17 12:28:32,854 INFO    MainThread:45364 [wandb_run.py:_finish():2154] finishing run suspicious-cow-self/Reddit_Sentiment_Analysis/cbtejz2q
2024-08-17 12:28:32,854 INFO    MainThread:45364 [wandb_run.py:_atexit_cleanup():2418] got exitcode: 0
2024-08-17 12:28:32,855 INFO    MainThread:45364 [wandb_run.py:_restore():2400] restore
2024-08-17 12:28:32,855 INFO    MainThread:45364 [wandb_run.py:_restore():2406] restore done
2024-08-17 12:28:35,095 INFO    MainThread:45364 [wandb_run.py:_footer_history_summary_info():4076] rendering history
2024-08-17 12:28:35,095 INFO    MainThread:45364 [wandb_run.py:_footer_history_summary_info():4108] rendering summary
2024-08-17 12:28:35,104 INFO    MainThread:45364 [wandb_run.py:_footer_sync_info():4035] logging synced files
